workers/ohsome_quality_analyst/utils/definitions.py

```diff
@@ -208,5 +207,25 @@ def get_report_classes() -> Dict:
     )
 
 
-ReportResult = collections.namedtuple("Result", "label value text")
-ReportMetadata = collections.namedtuple("Metadata", "name description")
+def get_indicator_names() -> List[str]:
+    return list(load_metadata("indicators").keys())
+
+
+def get_report_names() -> List[str]:
+    return list(load_metadata("reports").keys())
+
+
+def get_layer_names() -> List[str]:
+    return list(load_layer_definitions().keys())
+
+
+def get_dataset_names() -> List[str]:
+    return list(DATASETS.keys())
+
+
+def get_fid_fields() -> List[str]:
+    fid_fields = []
+    for _, dataset in DATASETS.items():
+        fid_fields.append(dataset["default"])
+        fid_fields += list(dataset["other"])
```

Comment:
This is prone to break in the future. E.g. no "other" fid fields available then the "default" ones. This will lead to KeyError.

Reply:
Changed to check for the existence of the key.